src/main/java/com/jcabi/log/VerboseProcess.java

```diff
@@ -420,38 +451,20 @@ private static void close(final Closeable res) {
             this.output = out;
             this.level = lvl;
         }
+
         @Override
         public Void call() throws Exception {
             final BufferedReader reader = new BufferedReader(
                 Channels.newReader(
-                    Channels.newChannel(this.input),
-                    VerboseProcess.UTF_8
+                    Channels.newChannel(this.input), VerboseProcess.UTF_8
                 )
             );
             try {
                 final BufferedWriter writer = new BufferedWriter(
                     new OutputStreamWriter(this.output, VerboseProcess.UTF_8)
                 );
                 try {
-                    while (true) {
-                        if (Thread.interrupted()) {
-                            Logger.debug(
-                                VerboseProcess.class,
-                                "explicitly interrupting read from buffer"
-                            );
-                            break;
-                        }
-                        final String line = reader.readLine();
-                        if (line == null) {
-                            break;
-                        }
-                        Logger.log(
-                            this.level, VerboseProcess.class,
-                            ">> %s", line
-                        );
-                        writer.write(line);
-                        writer.newLine();
-                    }
+                    logFromReader(reader, writer);
```

Comment:
@dean-e-clark it's instance method, our style demands to put `this.` to qualify the call